frontend/src/pages/meditation/components/meditation-modal/meditation-modal.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  Button,
+  Input,
+  InputFile,
+  Modal,
+} from '#libs/components/components.js';
+import { EMPTY_ARRAY_LENGTH } from '#libs/constants/constants.js';
+import { DataStatus } from '#libs/enums/enums.js';
+import {
+  useAppForm,
+  useAppSelector,
+  useCallback,
+  useEffect,
+} from '#libs/hooks/hooks.js';
+import {
+  createMeditationEntryValidationSchema,
+  type MeditationEntryCreateForm,
+} from '#packages/meditation/meditation.js';
+import { DEFAULT_MEDITATION_PAYLOAD } from '#pages/meditation/libs/constants/constants.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  reference: React.RefObject<HTMLDialogElement>;
+  onSubmit: (title: string, file: File) => void;
+};
+
+const MeditationModal: React.FC<Properties> = ({ reference, onSubmit }) => {
+  const { meditationDataStatus } = useAppSelector(({ meditation }) => {
+    return {
+      meditationDataStatus: meditation.meditationEntriesDataStatus,
+    };
+  });
+  const { control, errors, handleSubmit, reset } =
+    useAppForm<MeditationEntryCreateForm>({
+      defaultValues: DEFAULT_MEDITATION_PAYLOAD,
+      validationSchema: createMeditationEntryValidationSchema,
+    });
+
+  const hasError = Boolean(Object.keys(errors).length > EMPTY_ARRAY_LENGTH);
```

Comment:
it's already boolean
```suggestion
  const hasError = Object.keys(errors).length > EMPTY_ARRAY_LENGTH;
```

Reply:
Done